src/modelbench/run.py

```diff
@@ -190,18 +191,27 @@ def consistency_check(journal_path, verbose):
 
 
 def find_suts_for_sut_argument(sut_uids: List[str]):
```

Comment:
Maybe this function should be named `find_suts_for` to match other functions?

Reply:
Which other functions?